dev-python/matplotlib/matplotlib-3.7.1.recipe

```diff
@@ -50,37 +49,46 @@ PROVIDES_python39="
 	"
 REQUIRES_python39="
 	cmd:python$pythonVersion
-	matplotlib$secondaryArchSuffix
-	dateutil_$pythonPackage
-	numpy${secondaryArchSuffix}_$pythonPackage
-	pillow${secondaryArchSuffix}_$pythonPackage
-	kiwisolver${secondaryArchSuffix}_$pythonPackage
-	pytz_$pythonPackage
-	six_$pythonPackage
+	cycler_$pythonPackage
+	numpy_$pythonPackage
+	pybind11_$pythonPackage
+	kiwisolver_$pythonPackage
+	pyparsing_$pythonPackage
+	setuptools_scm_$pythonPackage
```

Comment:
pybind11 and setuptools_scm removed, on the install requires, setuptools doesn't put the correct version in the egg info file, and you'll get a notice about scm when building with setuptools instead of setuptools_scm.

As for the missing ones, so far no complaints on importing the module with python3. Could have a check though what's missing.

Reply:
Looking at contourpy I'm running into it not finding pybind11:
```
* Building wheel...
+ meson setup /sources/contourpy-1.1.0 /sources/contourpy-1.1.0/.mesonpy-yfix093y/build -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md --vsenv --native-file=/sources/contourpy-1.1.0/.mesonpy-yfix093y/build/meson-python-native-file.ini
The Meson build system
Version: 1.1.0
Source dir: /sources/contourpy-1.1.0
Build dir: /sources/contourpy-1.1.0/.mesonpy-yfix093y/build
Build type: native build
Project name: contourpy
Project version: 1.1.0
C++ compiler for the host machine: c++ (gcc 2 .95 .0 "2.95.3-haiku-2017_07_20")
C++ linker for the host machine: c++ ld.bfd 2.17-haiku
Host machine cpu family: x86
Host machine cpu: bepc
Program python found: YES (/bin/python3.9)
Found pkg-config: /bin/pkg-config (0.29.2)
pybind11-config found: NO
Found CMake: /bin/cmake (3.24.2)
Run-time dependency pybind11 found: NO (tried pkgconfig, config-tool and cmake)

../../meson.build:23:15: ERROR: Dependency "pybind11" not found, tried pkgconfig, config-tool and cmake

A full log can be found at /sources/contourpy-1.1.0/.mesonpy-yfix093y/build/meson-logs/meson-log.txt

ERROR Backend subprocess exited when trying to invoke build_wheel
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 1.
cleaning 'chroot/boot' folder
keeping chroot folder /boot/home/haikuports/dev-python/contourpy/work-1.1.0 intact for inspection
Error: Build has failed - stopping.
```
Seems like it's looking for `pybind11-config` which we don't provide? ;)

Reply:
fonttools update PR: https://github.com/haikuports/haikuports/pull/8887 (there was no python3.9 package, set that as defaultVersion)

Reply:
Haven't checked yet, but as you wrote this it came to mind that we do have a python module "pkgconfig", maybe this could help out here?

Reply:
Thanks!